### PR TITLE
Add TNMTMISTYFANLTTSSYCREANKNOWSBIYHWYP v1.1.0

### PR DESCRIPTION
**Pull request by Psychpsyo**
Added new mod version release

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -5470,6 +5470,10 @@
                         {
                             "url": "https://github.com/Psychpsyo/TNMTMISTYFANLTTSSYCREANKNOWSBIYHWYP/releases/download/1.0.0/TheNeosModThatMakesItSoThatYourFingersAreNoLongerTrustedTouchSourcesSoYouCanRestEasyAtNightKnowingNoOneWillShoveButtonsIntoYourHandWithoutYourPermission.dll",
                             "sha256": "63445a9255dd8e3c3e33cfa99e9bd6878ed5f847e00f6503f957e6e02987553e"
+                        },
```
Comment (XDelta) [inline manifest.json:5473]: This should be a new version, not an artifact to the existing version

Comment (Psychpsyo) [reply manifest.json]: Fixed now.

File: manifest.json [hunk 2]
```diff
@@ -5472,6 +5472,15 @@
                             "sha256": "63445a9255dd8e3c3e33cfa99e9bd6878ed5f847e00f6503f957e6e02987553e"
                         }
                     ]
+                },
+                "1.1.0": {
+                    "releaseUrl": "https://github.com/Psychpsyo/TNMTMISTYFANLTTSSYCREANKNOWSBIYHWYP/releases/tag/1.1.0",
```
Comment (XDelta) [inline manifest.json:5477]: releaseUrl is a 404

**Review by XDelta — CHANGES_REQUESTED** [review]

File: manifest.json (hunk 2 above)
Comment (Psychpsyo) [reply manifest.json]: Resolved as well.

**Review by XDelta — APPROVED** [review]

**Pull request merged**